web/src/resources/product/views/CreateProduct.test.jsx

```diff
@@ -0,0 +1,158 @@
+// CreateProduct.test.jsx
```

Comment:
minor comment, but it feels like this muddies up the folders a little bit. maybe a new folder for them, /components/views/tests/?

Reply:
I thought about that too. I'll update it.

Reply:
done